content/sensu-go/6.6/operations/deploy-sensu/generate-certificates.md

```diff
@@ -186,18 +188,15 @@ You should now have this set of files for each backend:
 filename               | description                  | required on backend?|
 -----------------------|------------------------------|---------------------|
 `backend-*.pem`        | Backend server certificate   | {{< check >}}       |
-`backend-*-key.pem`    | Backend server private key   | {{< check >}}       |
```

Comment:
Just noting this because I'm not certain whether `backend-*-key.pem` is the same as (or copied from) `ca-key.pem`

I'm not sure where else it could have come from, so I guessed it should be deleted, but we have it marked as required.

Reply:
@hillaryfraley backend-key.pem is required by the backend's `key-file` param